include/inviwo/core/properties/cameraproperty.h

```diff
@@ -95,11 +95,11 @@ class IVW_CORE_API CameraProperty : public CompositeProperty, public TrackballOb
     CameraProperty& setCamera(const std::string& cameraIdentifier);
     CameraProperty& setCamera(std::unique_ptr<Camera> camera);
 
-    virtual const vec3& getLookFrom() const override;
+    virtual vec3 getLookFrom() const override;
```

Comment:
Why?

Reply:
The point light source processor has a trackball implementation and needs to return a look from position. This position is coming from the PositionProperty and are computed on-the-fly. Hence, no reference.